src/java/picard/sam/markduplicates/util/OpticalDuplicateFinder.java

```diff
@@ -87,20 +86,41 @@ public OpticalDuplicateFinder(final String readNameRegex, final int opticalDupli
 
         final PhysicalLocation actualKeeper = keeperOrNull(list, keeper);
 
+        final Log log = Log.getInstance(OpticalDuplicateFinder.class);
+        ProgressLogger progressLogger = new ProgressLogger(log, 10000, "examined", "ReadEnds to keeper");
+        if(length > 1000) {
```

Comment:
thanks for the comments @nh13. to be clear, you want the 1000 value to be a parameter to the method? Would it be OK if I make it a private member of the class with a setter and default 1000?